e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_unreads_spec.js

```diff
@@ -81,15 +81,12 @@ describe('Keyboard Shortcuts', () => {
         const team1 = teamAndChannels[0].team;
         const team1Channels = teamAndChannels[0].channels;
 
-        // cy.get('@team1').then((team1) => {
         cy.visit(`/${team1.name}/channels/town-square`);
 
         // # Type CTRL/CMD+K
         cy.get('#post_textbox').cmdOrCtrlShortcut('K');
 
         // # Verify that the mentions in the channels of this team are displayed
-        cy.get('#suggestionList').should('exist').children().should('have.length', count);
```

Comment:
Removed this test since suggestion item is being verified individually on succeeding steps.